src/colvargrid.h

```diff
@@ -416,6 +416,20 @@ template <class T> class colvar_grid : public colvarparse {
     return (int) cvm::floor( (value.real_value - lower_boundaries[i].real_value) / widths[i] );
   }
 
+  /// \brief Report the fraction of bin beyond current_bin_scalar()
+  inline cvm::real current_bin_scalar_fraction(int const i) const
+  {
+    return value_to_bin_scalar_fraction(use_actual_value[i] ? cv[i]->actual_value() : cv[i]->value(), i);
+  }
+
+  /// \brief Use the lower boundary and the width to report which bin
+  /// the provided value is in
```

Comment:
Update comment?

Reply:
I'm not sure what mean here. Is the comment for "current_bin_scalar_fraction" not clear?

Reply:
I mean the comment of `value_to_bin_scalar_fraction()` not in sync with the one of `current_bin_scalar_fraction()`. The comment doesn't change behavior so I'll merge anyway and update later.